canonical-websites/snapcraft.io
# active devices, eslint and code refactor

## Summary

- Added active devices graph to measure page
- Added eslint to yarn test
- Refactored some of the es6 code

implements design from https://github.com/CanonicalLtd/snapcraft-design/issues/116

## QA

- Checkout the branch
- `./run test` - make sure the tests work
- `./run serve`
- http://0.0.0.0:8004/snaps/dwarf-fortress/measure or http://snapcraft.io-pr-197.run.demo.haus/snaps/dwarf-fortress/measure
- Check both graphs work as expected
